app/controllers/image-set.js

```diff
@@ -0,0 +1,73 @@
+import Ember from 'ember';
+import {imageTypes, genders} from 'lion-guardians/utils/units';
+
+export default Ember.Controller.extend({
+  imageTypes: imageTypes,
+  genders: genders,
+  uploadImageType: imageTypes[0],
+  uploadIsPublic: false,
+  isEditing: false,
+
+  organizations: function() {
+    return this.store.findAll('organization');
```

Comment:
This is an async call (and `findAll` is not the preferred way to "find all" anymore. Just a `store.find('organization')` is the preferred way to find all instances of a model type now), so this property will be a promise proxy, which may not be what you want. I am surprised the select works when doing it this way, actually.

I would load the organizations in a route somewhere instead. Less likely to have async inconsistencies that way.